subgraph/src/fleek-nfa.ts

```diff
@@ -80,6 +82,52 @@ export function handleCollectionRoleGranted(
   entity.transactionHash = event.transaction.hash;
 
   entity.save();
+
+  if (event.params.role === 0) {
```

Comment:
I would recommend starting to move these into separate files.  Otherwise this one file is going to get large and have merge issues etc.  Maybe we can start creating "epic" level folders here like `accessControl`, `mint`, etc and putting the handlers in the respective ones. Like this would go under access control.